source/assets/javascripts/_gradient-background.js

```diff
@@ -1,7 +1,33 @@
-$(document).mousemove(function(e){
-  var $width = ($(document).width())/255;
-  var $height = ($(document).height())/255;
-  var $pageX = parseInt(e.pageX / $width,10);
-  var $pageY = parseInt(e.pageY / $height,10);
-  $("body").css("background", "linear-gradient(135deg, rgb(112, "+$pageX+", "+$pageY+"), rgb(255, "+$pageY+", "+$pageX+")" );
+$(function() {
+  var lastXPosition, lastYPosition;
+
+  $(document).mousemove(function(e){
+    if(lastXPosition == e.clientX && lastYPosition == e.clientY) {
+      return
+    }
+
+    lastXPosition = e.clientX
+    lastYPosition = e.clientY
+
+    console.log(e)
+    var $width = ($(document).width())/255;
+    var $height = ($(document).height())/255;
+    var $pageX = parseInt(e.clientX / $width,10);
+    var $pageY = parseInt(e.clientY / $height,10);
+    var gradient = "linear-gradient(135deg, rgb(112, "+$pageX+", "+$pageY+"), rgb(255, "+$pageY+", "+$pageX+")"
+
+    $(".mobile-device").css("background-image", gradient);
+
+    console.log($pageX);
+  });
 });
+
+function hexToRgb(h)
+{
+    var r = parseInt((cutHex(h)).substring(0,2),16), g = ((cutHex(h)).substring(2,4),16), b = parseInt((cutHex(h)).substring(4,6),16)
+    return r+''+b+''+b;
+}
+function cutHex(h) {return (h.charAt(0)=="#") ? h.substring(1,7):h}
+
+//usage
+console.log(hexToRgb("#FFFFFF"));
```

Comment:
This was me messing with trying to get things to work in Safari. I'll probably delete it and let it degrade gracefully.